release_notes/ocp-4-4-release-notes.adoc

```diff
@@ -3785,3 +3785,35 @@ This is because the service CA is automatically rotated as of {product-title} 4.
 
 After this one-time manual restart, subsequent upgrades and rotations will ensure restart before the service CA expires without requiring manual intervention.
 ====
+
+[[ocp-4-4-30]]
+=== RHBA-2020:4321 - {product-title} 4.4.30 Bug Fix Update
+
+Issued: 2020-11-11
+
+{product-title} release 4.4.30 is now available. The list of container images and bug fixes included in the update are documented in the link:https://access.redhat.com/errata/RHBA-2020:4321[RHBA-2020:4321] advisory. The RPM packages included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2020:4322[RHBA-2020:4322] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:
+
+link:https://access.redhat.com/solutions/5551251[{product-title} 4.4.30 container image list]
+
+[[ocp-4-4-30-upgrading]]
+==== Upgrading
+
+To upgrade an existing {product-title} 4.4 cluster to this latest release, see xref:../updating/updating-cluster-between-minor.adoc#update-upgrading-web_updating-cluster-between-minor[Updating a cluster by using the web console] for instructions.
+
+[IMPORTANT]
+====
+If you are upgrading to this release from {product-title} 4.3.3 or earlier, you must restart all Pods after the upgrade is complete.
```

Comment:
(With our new guidance, we are lowercasing "pod" now but maybe it's good to leave it capped here to be consistent.)

Reply:
Could go ahead and fix them all. @codyhoag thoughts on this retroactive fix in the RNs?

Reply:
Yeah, makes sense to me :+1: